### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by eerapheal**
Complete desktop version.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,21 +18,33 @@
...
+        <div></div>
+      </div>
+      <div id="bar">
+        <ul>
+          <li><a href="#">Portfolio</a></li>
+         <li><a href="#">About</a></li>
+         <li><a href="#">Contact</a></li>
+        </ul>
+        <a href="#"><img src="image/Icon-Mail.png" alt="mail"></a>
+      </div>
```
Comment (ShahierNashaat) [inline index.html:35]: - [ ] Kindly be noticed that you should follow the Figma design in your portfolio.
- [ ] Please in the desktop size make the font bigger, add the background image, make the icons of the nav bar to the right, and make the icons for Twitter, Linkedin, etc a little bit up.
- [ ] Also in the mobile size make the nav bar as the Figma design.

Your Design | Figma Design
---------------- | --------------------
Screenshot from 2022-07-14 00-06-57 | Screenshot from 2022-07-14 00-06-31

Your Design | Figma Design
---------------- | --------------------
Screenshot from 2022-07-14 00-11-59 | Screenshot from 2022-07-14 00-11-33

File: index.html [hunk 2]
```diff
@@ -44,24 +56,29 @@ <h2 class="sub-headerline">I’am a Software Developer</h2>
...
+        </p>
         <ul class="grid1">
           <li>css</li>
           <li>html</li>
           <li>bootsrap</li>
           <li>ruby</li>
         </ul>
         <button class="bnt-top" type="button">See projests</button>
+       </div>
       </div>
```
Comment (ShahierNashaat) [inline index.html:78]: - [ ] Kindly be noticed that should follow the Figma design in your portfolio.
- [ ] Please put the first project in the center as it is at the left of the page.
- [ ] Also please make the div that holds each other projects bigger than the Figma design.
- [ ] Also in the mobile size make the see project button width the same as the width of the div that holds the project.

Your Design | Figma Design
---------------- | ------------------
Screenshot from 2022-07-14 00-08-11 | Screenshot from 2022-07-14 00-07-45

Your Design | Figma Design
---------------- | ------------------
Screenshot from 2022-07-14 00-08-52 | Screenshot from 2022-07-14 00-08-33

Your Design | Figma Design
---------------- | ------------------
Screenshot from 2022-07-14 00-12-38 | Screenshot from 2022-07-14 00-12-26

File: index.html [hunk 3]
```diff
@@ -144,67 +161,71 @@ <h4 class="grid-2h4">Profesional Art Printing Data</h4>
...
     <section class="sec-2">
-      <div class="mrre">
-        <div class="sec-3-div">
+      <div class="mrre bckimg">
+        <div class="sec-3-div ">
           <h2 class="sec-3-div-h1">About me</h2>
           <p class="sec-3-div-p">Hello I’m a software developer! I can help you build a product , feature or website
             Look through some of my work and experience! If you like what you see and have a project you need coded,
             don’t hestiate to contact me.</p>
           <button class="sec-3-div-btn" type="button">Get My Resume</button>
```
Comment (ShahierNashaat) [inline index.html:174]: - [ ] Kindly be noticed that you should follow the Figma design in your portfolio.
- [ ] Kindly make the background image of the about me section in the mobile size a little bit down.
- [ ] Also make the `Get My Resume` button a the left of the page.

Your Design | Figma Design
---------------- | --------------------
Screenshot from 2022-07-14 00-13-22 | Screenshot from 2022-07-14 00-13-05

File: index.html [hunk 4]
```diff
@@ -144,67 +161,71 @@ <h4 class="grid-2h4">Profesional Art Printing Data</h4>
...
     </section>
     <section class="Contact-Me">
       <article class="form-container">
         <h2 class="form-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in
-        touch.</h2>
+        touch.
+        </h2>
         <form class="contact-form" action="https://formspree.io/f/mknylokq" method="post">
             <input class="name-input" type="text" name="name" placeholder="Name" maxlength="30" required>
             <div class="line3"></div>
```
Comment (ShahierNashaat) [inline index.html:231]: - [ ] Kindly be noticed that you should follow the Figma design in your portfolio.
- [ ] Please delete the extra space between the form and the footer.

Your Design | Figma Design
---------------- | --------------------
Screenshot from 2022-07-14 00-09-58 | Screenshot from 2022-07-14 00-09-41

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @eerapheal , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @eerapheal , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights


## Required Changes ♻️

- [ ]  I can see you've done some changes, but again you still miss important part of the project as previous reviewer suggested. 
Please refer to the figma design, and impliment all the changes requested by the reviewer. 

Your design 
image

Figma design 
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @eerapheal, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Almost near

 ## Highlights
✔️ All linter checks pass
✔️ Descriptive pull request title

 ## Required Changes ♻️

- [ ] Kindly make the changes as stated by the previous reviewer and make your design match the Figma design by giving appropriate padding and margin.

| Your design  | Figma design|
| ------------- | ------------- |
| image  | image |


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @eerapheal, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## ❌ Invalid Code Review Request

You have submitted a project, that has some failing linter checks. Kindly fix the linter errors and submit again. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @eerapheal , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## ❌ Invalid Code Review Request

 You have submitted a project, that has some failing linter checks. Kindly fix the linter errors and submit again.
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by eerapheal** [comment]
There is no linters message at my end.

File: index.html [hunk 5]
```diff
@@ -18,60 +18,79 @@
...
     <section class="section-fir">
       <div class="headline-div">
-        <h1 class="headerline">Hey There. I’m Raphael</h1>
-        <h2 class="sub-headerline">I’am a Software Developer</h2>
+        <h1 class="headerline">Hey There. I’m Ralph.</h1>
+        <h2 class="sub-headerline">I’am a Software Developer,</h2>
         <p class="content-p">I can help you build a product , feature or website Look through some of my work and
           experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
       </div>
-      <div>
```
Comment (mwanawabangona) [inline index.html:35]: - [ ] `optional`
- [ ] please add the background image on the desktop version. please see screenshots below

`figma template`
Screenshot from 2022-07-17 02-38-44

`your template`
Screenshot from 2022-07-17 02-38-14

**Review by mwanawabangona — APPROVED** [review]
Hi @eerapheal ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eerapheal** [comment]
I really appreciate your comments

**Pull request merged**